tests/Resources/Admin/ContentAdmin.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2017 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DoctrinePHPCRAdminBundle\Tests\Resources\Admin;
+
+use Doctrine\Bundle\PHPCRBundle\Form\DataTransformer\DocumentToPathTransformer;
+use Doctrine\Common\Persistence\ManagerRegistry;
+use Sonata\AdminBundle\Datagrid\DatagridMapper;
+use Sonata\AdminBundle\Datagrid\ListMapper;
+use Sonata\AdminBundle\Form\FormMapper;
+use Sonata\CoreBundle\Form\Type\CollectionType;
+use Sonata\DoctrinePHPCRAdminBundle\Admin\Admin;
+use Sonata\DoctrinePHPCRAdminBundle\Tests\Resources\Document\Content;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+
+/**
+ * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
+ */
+class ContentAdmin extends Admin
+{
+    /**
+     * @var ManagerRegistry
+     */
+    private $managerRegistry;
+
+    public function setManagerRegistry(ManagerRegistry $managerRegistry)
+    {
+        $this->managerRegistry = $managerRegistry;
+    }
+
+    public function getExportFormats()
+    {
+        return array();
+    }
+
+    public function toString($object)
+    {
+        return $object instanceof Content && $object->getTitle()
+            ? $object->getTitle()
+            : $this->trans('link_add', array(), 'SonataAdminBundle');
+    }
+
+    protected function configureListFields(ListMapper $listMapper)
+    {
+        $listMapper
+            ->addIdentifier('id')
+            ->add('title');
+    }
+
+    protected function configureFormFields(FormMapper $formMapper)
+    {
+        $formMapper
+            ->with('form.group_general')
+            ->add('name', 'Symfony\Component\Form\Extension\Core\Type\TextType')
+            ->add('title', 'Symfony\Component\Form\Extension\Core\Type\TextType')
+            ->add(
+                'children',
+                CollectionType::class,
+                array('label' => false, 'type_options' => array('delete' => true, 'delete_options' => array('type' => CheckboxType::class, 'type_options' => array('required' => false, 'mapped' => false)))),
```

Comment:
I think the default type for the `_delete` field set to a form of type `AdminType` has to be set to `CheckboxType` not to be removed. 
btw: here is my current setup:
```
maximilian@maximilian-ThinkPad-T540p:~/OpenSource/Cmf/SonataDoctrinePhpcrAdminBundle$ composer show sonata-project/*
You are running composer with xdebug enabled. This has a major impact on runtime performance. See https://getcomposer.org/xdebug
sonata-project/admin-bundle    3.20.1          The missing Symfony Admin Generator
sonata-project/block-bundle    3.3.2           Symfony SonataBlockBundle
sonata-project/cache           1.0.7           Cache library
sonata-project/core-bundle     3.x-dev b1629c5 Symfony SonataCoreBundle
sonata-project/datagrid-bundle 2.2.1           Symfony SonataDatagridBundle
sonata-project/exporter        1.7.1           Lightweight Exporter library
maximilian@maximilian-ThinkPad-T540p:~/OpenSource/Cmf/SonataDoctrinePhpcrAdminBundle$ composer show symfony/*
You are running composer with xdebug enabled. This has a major impact on runtime performance. See https://getcomposer.org/xdebug
symfony/monolog-bundle    v3.1.0 Symfony MonologBundle
symfony/phpunit-bridge    v3.3.6 Symfony PHPUnit Bridge
symfony/polyfill-intl-icu v1.4.0 Symfony polyfill for intl's ICU-related data and classes
symfony/polyfill-mbstring v1.4.0 Symfony polyfill for the Mbstring extension
symfony/polyfill-php56    v1.4.0 Symfony polyfill backporting some PHP 5.6+ features to lower PHP versions
symfony/polyfill-php70    v1.4.0 Symfony polyfill backporting some PHP 7.0+ features to lower PHP versions
symfony/polyfill-util     v1.4.0 Symfony utilities for portability of PHP codes
symfony/security-acl      v3.0.0 Symfony Security Component - ACL (Access Control List)
symfony/symfony           v3.3.6 The Symfony PHP framework
```